prime-router/src/main/kotlin/fhirengine/engine/Message.kt

```diff
@@ -58,6 +58,7 @@ data class RawSubmission(
     val blobURL: String,
     val digest: String,
     val blobSubFolderName: String,
+    val topic: String,
```

Comment:
I think we might also want a follow on ticket that captures cleaning up after we're confident there are no messages left on the queue without a topic?

Reply:
Good call! Given our discussion of not needing to worry about the transition as much, I went with `Topic` instead of `Topic?`, let me know if that seems okay.